lib/linguist/languages.yml

```diff
@@ -5169,7 +5169,9 @@ Vim script:
   - viml
   - nvim
   extensions:
+  - ".vba"
```

Comment:
The first extension should be the "primary extension", only subsequent extensions need to be sorted alphabetically. `.vim` is the primary extension for Vim scripts, no?

Reply:
Indeed, `.vim` is still the primary extension. `.vba` and `.vmb` are Vimball archives (Vim plugins archived by [Vimball](https://www.vim.org/scripts/script.php?script_id=1502)). They are Vim scripts, but they will work only if the Vimball script is installed. They invoke it with `UseVimball` to extract the contents, which are in the rest of the archive.

AFAIK all Vimball archives have this `UseVimball` call. And if something has `.vba` and `.vmb` and it's not a Vimball archive, it's very unlikely that it's a different kind of Vim script.